Assets/Scripts/UsedInTutorials/PlayerCharacter.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerCharacter : MonoBehaviour {
+
```

Comment:
#region all these serialize fields, it allows you to collapse it like you can a function, it's pretty nice, trust me. Code and Unity style conventions document has the info on that for you